common/static/js/vendor/pdfjs/viewer.html

```diff
@@ -108,9 +108,9 @@
               <span data-l10n-id="download_label">Download</span>
             </button>
 
-            <a href="#" id="secondaryViewBookmark" class="secondaryToolbarButton bookmark visibleSmallView" title="Current view (copy or open in new window)" tabindex="55" data-l10n-id="bookmark">
+            <button type="button" id="secondaryViewBookmark" class="secondaryToolbarButton bookmark visibleSmallView" title="Current view (copy or open in new window)" tabindex="55" data-l10n-id="bookmark">
```

Comment:
This is a vendor file, and it does not appear we have edited it previously. I would prefer not editing vendor files if at all possible.